data/json/flags.json

```diff
@@ -2223,5 +2223,17 @@
     "id": "REBREATHER_CART",
     "type": "json_flag",
     "info": "A filter for a rebreather."
+  },
+  {
+    "id": "panoramic_mask_outsert",
```

Comment:
Flags should be in allcaps.

~~Also I think you could shorten flag name to PANORAMIC_INSERT/_OUTSERT~~

Reply:
Yep, and I completely forgot to make them in all caps. Fixing now. Thanks for pointing it out.